godaddy/metadata.csv

```diff
@@ -1 +1,15 @@
-metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
\ No newline at end of file
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
+godaddy.certificate.canceled_certificates,count,,,,The total number of canceled certificates,0,godaddy,Canceled Certificates,,
+godaddy.certificate.certificate_expiration_days,count,,day,,Count of expiration days by certificate,0,godaddy,Certificate Expiration Days,,
+godaddy.certificate.certificate_status,count,,,,Contains the certificate with status,0,godaddy,Certificate Status,,
```

Comment:
"Contains the certificate with status" is unclear to me. It seems this is a "count" metric -- what are we counting? What are the possible statuses?

Reply:
This metric contains the certificate ID, and its current status. We use this metric to trigger monitors based on changes in certificate status. Both the certificate ID and status are tags, we are not really counting anything here but using the tags. The certificate status can be one of the following: (ISSUED, CANCELED, DENIED, EXPIRED, PENDING_ISSUANCE, PENDING_REKEY, PENDING_REVOCATION, REVOKED, UNUSED). Updated the description a bit let me know if this needs a change.